backend/test/services/core_data.py

```diff
@@ -22,6 +25,13 @@ def setup_insert_data_fixture(session: Session):
     permission_data.insert_fake_data(session)
     organization_test_data.insert_fake_data(session)
     event_test_data.insert_fake_data(session)
+    room_data.insert_fake_data(session)
+    operating_hours_data.insert_fake_data(session, time.time_data())
```

Comment:
The only piece causing a lot of errors here was getting the `time` fixture to be recognized. Despite importing it in this file and numerous other files, there was still an error complaining about "missing parameter `time`". The only way for me to get around this was manually passing in `.time_data()` into these functions directly. Thoughts @KrisJordan?